src/app/containers/ATIAnalytics/index.jsx

```diff
@@ -17,9 +17,10 @@ const ATIAnalytics = ({ data }) => {
 
   const pageTypeHandlers = {
     article: buildArticleATIUrl,
-    frontPage: buildFrontPageATIUrl,
+    frontPage: buildIndexPageATIUrl,
```

Comment:
I think this should just remain as it was since we have an IDX key which points to the idxPageATIUrl

Reply:
Pointed both frontPage and IDX to buildIndexPageATIUrl as the functionality was pretty much the same for both pages, except for [one thing](https://github.com/bbc/simorgh/blob/6f713379bc90fe4409b5a3f7a0a87b172662f864/src/app/containers/ATIAnalytics/params/indexPage/buildParams.js#L26).

Reply:
Having it as before would mean we have 2 params modules with code that is identical, except for one thing

Reply:
I think the best approach would be to have a shared function which accepts `contentType` as a parameter. You can out this function in a utility file.

Reply:
I have created a [getContentType](https://github.com/bbc/simorgh/blob/1082e46dcb7bbefff7d8e3dae4377167f780e3e9/src/app/lib/analyticsUtils/indexPage/index.js#L46) utility function